addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -49,6 +50,18 @@ class ChannelMember(models.Model):
     rtc_session_ids = fields.One2many(string="RTC Sessions", comodel_name='discuss.channel.rtc.session', inverse_name='channel_member_id')
     rtc_inviting_session_id = fields.Many2one('discuss.channel.rtc.session', string='Ringing session')
 
+    @api.autovacuum
+    def _gc_unpin_outdated_sub_channels(self):
+        members = self.env["discuss.channel.member"].search(
+            [
+                ("channel_id.parent_channel_id", "!=", False),
+                ("last_interest_dt", "<", fields.Datetime.now() - timedelta(days=2)),
```

Comment:
Only the last interest of the member?
I think this should take the last interest of the channel into account too.
If new messages are received, it updates the channel instead.

Reply:
Updated as well as the test :+1: